core/src/main/java/com/redhat/lightblue/client/request/data/DataDeleteRequest.java

```diff
@@ -31,6 +31,7 @@ public JsonNode getBodyJson() {
         if (queryExpression != null) {
             node.set("query", queryExpression.toJson());
         }
+        appendExecutionToJson(node);
```

Comment:
It's done similar to how range was implemented.  Should clean up both if this change is implemented.